mysqlnd_azure.c

```diff
@@ -387,6 +433,9 @@ MYSQLND_METHOD(mysqlnd_azure_data, connect)(MYSQLND_CONN_DATA ** pconn,
 
             //init redirect_conn options failed, just use the proxy connection
             if (ret == FAIL) {
+              if (MYSQLND_AZURE_G(logLevel) > 0) {
+                AZURE_LOG("ERROR", "Set redirection information to redirect conn failed.");
```

Comment:
here is to check whether the operation of copying options  from direct conn object to redirect conn object failed. So maybe we can use the same mssage for debug, i.e. "Init redirect_conn options failed"